client/src/App.jsx

```diff
@@ -18,15 +15,17 @@ import LandingPage from './Components/LandingPage.jsx';
 import TrendsPage from './Components/TrendsPage.jsx';
 import LoginPage from './Components/LoginPage.jsx';
 import ProfilePage from './Components/ProfilePage.jsx';
-import Budget from './Components/BudgetPage.jsx';
+// import Budget from './Components/BudgetPage.jsx';
```

Comment:
Can you swap this out for BudgetPage and/or just delete it?

Reply:
swapped out for BudgetPage